### DISCUSSION TITLE
Localize multi-site with different languages

### DISCUSSION THREAD

**Discussion by isedwards** [wagtail/wagtail-localize#380] [Q&A] [answered]
We have a Wagtail multi-site where each country publishes completely different content.

The Canada site uses English and French for their content whilst Switzerland make use of German, French and Italian. The sites in East Africa may have French and Swahili, or English and Swahili....

In a multi-site environment, is there a way to customise which langanges are used for each site?  If not, could you give pointers of  the easiest way to achieve this (would it require creating a fork)?

**Comment by kaedroho** [comment, marked as answer]
Hi @isedwards 

I think the only thing that's stopping you here is the way the "Sync from" locale feature is set up. It currently only allows you to select one locale to sync from and this applies to all sites. Open to ideas on how to improve this!

If you don't use this feature, you can create a separate homepage for each site (with that site's primary language. The "Add subpage" view at the root level has a dropdown in the header to allow you to set the language), and translate each site homepage into just the additional languages that the site needs.

**Reply by isedwards** [reply]
Thank you Karl, it's a shame to lose "sync from" (which is a great feature) - but this workaround will solve our immediate problem.

I guess a full solution would require `/admin/locales/2/` to have:

**Language: French**
**Sync from (default for all sites): English**  _(default choice could also be no sync)_

- [x] **Customise by site: Yes**

| site | sync from (dropdown) |
|-----|--------------------------|
| canada.example.com | <kbd> &nbsp;  &nbsp;  &nbsp; - &nbsp;  &nbsp;  &nbsp; </kbd> _(uses default from above, in this case English)_ |
| switzerland.example.com | <kbd> &nbsp; &nbsp;  German  &nbsp; &nbsp; </kbd> |
| uk.example.com | <kbd> &nbsp;  &nbsp; None &nbsp;  &nbsp; </kbd> _(no sync)_ |
